qucs-core/src/components/verilog/analogfunction.xml

```diff
@@ -1072,8 +1090,8 @@ double $(module)_d_$(function) (<admst:join select="variable[input='yes']" separ
 
 <admst:variable name="globalanalogfunction"/>
 
-<admst:open file="$(module).analogfunction.h">/*
- * $(module).analogfunction.h - analog function header
+<admst:open file="$(filename).$(module).analogfunction.h">/*
```

Comment:
this is just to avoid name clashes/file collisions. sure -- it is a crude workaround. a perfect implementation would not need either a header or even multiple headers for individual modules. we are not there yet.